docs/stac-api-spec/extensions/layer/README.md

```diff
@@ -0,0 +1,141 @@
+# STAC API Layer Extension
+
+**Extension [Maturity Classification](https://github.com/radiantearth/stac-api-spec/blob/master/extensions/README.md#extension-maturity): Proposal**
+
+This API extension defines the API resources and semantics for retrieving [STAC Layers](../../../stac-spec/extensions/layer/README.md)
+information defined for available STAC Items.
+
+## Methods
+
+| Path                    | Content-Type Header                                                                                         | Description                                                                       |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------- |
+| `GET /layers/`          | [Catalog](https://github.com/radiantearth/stac-api-spec/blob/master/stac-spec/catalog-spec/catalog-spec.md) | Catalog with links to layers.                                                      |
+| `GET /layers/{layerID}` | [`GeoJSON Feature`](https://github.com/azavea/stac4s/blob/4aba9c6691fde1d5235f165454ceaef6c6b3e165/docs/stac-spec/extensions/layer/json-schema/layer-schema.json)                                                                                       | [STAC Layer](../../../stac-spec/extensions/layer/README.md) as a GeoJSON Feature. |
+
+## Schema
+
+- [JSON Static Layer Schema](../../../stac-spec/extensions/layer/json-schema/layer-schema.json)
+
+## How It Works
+
+When this extension is implemented, the API supports `layers` endpoint that list all layers available. An example of the result output can be found [here](../../../stac-spec/extensions/layer/examples/landsat-stac-layers/layers/catalog.json).
```

Comment:
It would probably be good to say what a `layer` is for before talking about endpoints -- "a layer is a (potentially ad hoc) grouping/mosaic of items" or something. You have a good description in the README for the layer extension that it might be good to paraphrase here.